gcp/appengine/blog/content/posts/automating-and-scaling-vex-generation.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Automating and Scaling Vex Generation"
+date: 2023-03-03T16:46:51Z
+draft: true
+author: Brandon Lum
```

Comment:
are we able to add "Oliver Chang" and "Meder Kydyraliev" as well?

Reply:
We certainly can!